### ISSUE TITLE
[Feature Request]: Option to have Queue (currently labeled "Priority") management buttons on the menu/action bar.

### ISSUE THREAD

**Issue by enscott89** [VueTorrent/VueTorrent#1664]
### Description

I would absolutely love it if we had the option to have the buttons for sorting your queue on. Sadly the buttons are currently not there but rather are under the right click menu. I understand that some people wouldn't use them but I imagine that those who have queueing enabled would greatly appreciate the option.

### Describe the solution you'd like

Something like this mockup.
![mspaint_M0cGedkoxi](https://github.com/VueTorrent/VueTorrent/assets/38144053/2fdbf164-c23e-4b42-84ab-f87ab4988777)

### Describe alternatives you've considered

We could still use the right click menu. It's not exactly a deal-breaker.

### Complementary informations

- [X] Is this feature already implemented in the default WebUI?

**Comment by Larsluph** [comment]
I agree that it can be tedious to have to right click multiple times, but I'm not sure where it should go to look good on desktop and on mobile.
